website/docs/ref/react.doc.js

```diff
@@ -197,6 +197,28 @@ function renderButton(container: HTMLElement, visited?: boolean) {
 
   Any function that returns a React element can be used as a component class in
   a JSX expression.
+
+*/
+const AsyncGreeter = loadAsync(Greeter, new Promise((resolve, reject) => {
```

Comment:
This `loadAysnc` function is a bit mysterious and confusing. Is there a way we can make it a bit more clear what this is an example of?